checkov/common/runners/object_runner.py

```diff
@@ -20,6 +21,10 @@
 
 
 class Runner(BaseRunner):
+    def __init__(self, pbar: ProgressBar | None = None):
+        self.pbar = pbar if pbar else ProgressBar('')
```

Comment:
```suggestion
        self.pbar = pbar or ProgressBar('')
```
should do the same trick 😉